scripts/git_hooks/pre-commit

```diff
@@ -3,11 +3,11 @@
 ###################################
 ## Check OCaml Formatting
 
-make check-format
-if [ "$?" != 0 ]; then
-  echo '** REFUSING TO COMMIT: "$ make check-format" failed'
-  exit 1
-fi
+# make check-format
+# if [ "$?" != 0 ]; then
+#   echo '** REFUSING TO COMMIT: "$ make check-format" failed'
+#   exit 1
+# fi
```

Comment:
can you revert this change? (by the way, you can `git commit -n` to skip this step)